spylib/exceptions.py

```diff
@@ -2,12 +2,8 @@
 import logging
 
 
-class LoginException(Exception):
-    logging.debug("Login failed for an internal service request.")
-
-
-class RefreshException(Exception):
-    logging.debug("Refresh of access token failed for an internal service request.")
+class AuthCredentialException(Exception):
```

Comment:
Do we not want to provide some kind of default debugging for consumers?

Reply:
I wasn't too sure how that logging bit was triggered, does that get run at each instantiation of an exception?

Reply:
I believe so.

Probably this should have been done in a more modular fashion 🤔 It's fine leaving it for now perhaps! 

In the future it may make sense for us to setup some kind of module level logger that spylib uses 🤔